pkg/cmd/plugincheck2/main.go

```diff
@@ -136,8 +152,10 @@ func main() {
 	diags, err := runner.Check(
 		analyzers,
 		analysis.CheckParams{
+			ArchiveFile:           tmpZip.Name(),
```

Comment:
to validate the provenance we need unaltered plugin archive information and the source code reference.